keps/2724-topology-aware-scheduling/README.md

```diff
@@ -715,6 +772,12 @@ type PodSetTopologyRequest struct {
   // For example, in the context of JobSet this value is read from jobset.sigs.k8s.io/replicatedjob-replicas.
   SubGroupCount *int32
 
+  // PodSetGroup indicates the name of the group of PodSets to which this PodSet belongs to.
+  // PodSets with the same `PodSetGroup` should be assigned the same ResourceFlavor
+  //
+  // +optional
+  PodSetGroup *string `json:"podSetGroup,omitempty"`
```

Comment:
In the case of above LWS example, what is the expected name in this podSetGroup?

Reply:
It can be anything as long as it is the same value on other PodSets you would like to group. It will be set by `leaderworkerset_reconciler` (see here: https://github.com/kubernetes-sigs/kueue/pull/5878/files) to `leader-worker-set-group` if the annotation is present (if it is not, it will be left as nil)